src/Strings.js

```diff
@@ -0,0 +1,130 @@
+module.exports = function(steps, images, modulesInfo, addSteps, copy) {
+  // Genates a CLI string for the current sequence
+  var steps = steps
+  var images = images || {}
+
+  function toCliString() {
+    var cliStringSteps = `"`, cliOptions = {};
+    for (var step in this.steps) {
+      if (this.steps[step].options.name !== "load-image")
+        cliStringSteps += `${this.steps[step].options.name} `;
+      for (var inp in modulesInfo(this.steps[step].options.name).inputs) {
+        cliOptions[inp] = this.steps[step].options[inp];
+      }
+    }
+    cliStringSteps = cliStringSteps.substr(0, cliStringSteps.length - 1) + `"`;
+    return `sequencer -i [PATH] -s ${cliStringSteps} -d '${JSON.stringify(cliOptions)}'`
+  }
+
+  // Strigifies the current sequence
+  function toString(step) {
+    if (step) {
+      return stepToString(step);
+    } else {
+      return copy(this.images.image1.steps).map(stepToString).slice(1).join(',');
```

Comment:
I'd be really careful with the `this` reference. It is slippery! See above comment. 

Could you write a unit test for this code to test that it's working more narrowly, to eliminate that possibility? Just an idea.